quarkus/dist/src/main/content/bin/kc.bat

```diff
@@ -141,6 +149,11 @@ if not errorlevel == 1 (
     set IS_HELP_SHORT=true
 )
 
+if "%PRINT_ENV%" == "true" (
```

Comment:
Did not want to print `JAVA_OPTS` by default not to polute log.

Since it's now used mainly for debugging/testing purposes, using `PRINT_ENV` env var for now to enable it. If we find this useful, we might consider e.g. a documented `--verbose` switch.